app/views/claim_mailer/rejection_reasons/_early_career_payments.text.erb

```diff
@@ -0,0 +1,3 @@
+<% # TODO populate with appropriate content %>
```

Comment:
Can we remove this?

Reply:
@nomasprime have take a best guess on the content on this based on other journeys and what we have in the skeleton. Have removed. Will raise a query with the business to confirm exactly what is expected in this template